docs/get-started/connect/testnet.md

```diff
@@ -199,6 +236,17 @@ teku validator-client \
 Sepolia is a permissioned network and you can't run a validator client on it without [requesting to become a validator](https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg) first.
 
   </TabItem>
+
+  <TabItem value="Ephemery" label="Ephemery" default>
+
+```bash
+teku validator-client \
+    --network=ephemery                    \
+    --beacon-node-api-endpoint=<endpoint> \
+    --validator-keys=<path to key file>:<path to password file>[,<path to key file>:<path to password file>,...]
+```
+
+  </TabItem>
```

Comment:
maybe what we should do is leave out VC for now and just do this once it's supported. currently VC will not support the ephemery testnet resets..

we can raise a ticket to add ephemery reset support to the vc...

Reply:
Alright, thanks. I will remove it for now pending it is implemented.